crates/bitwarden-wasm-internal/src/vault/mod.rs

```diff
@@ -21,4 +23,8 @@ impl VaultClient {
     pub fn folders(&self) -> ClientFolders {
         ClientFolders::new(self.0.clone())
     }
+
+    pub fn totp(&self) -> ClientTotp {
+        ClientTotp::new(self.0.clone())
+    }
```

Comment:
thought: we might want to merge this client into some other bigger collection but hard to say what that might be right now. This is great for now 👍